### PR TITLE
bors: run tests when no `.rs` files were changed

### LINKED ISSUE #73: Fix bors script, so it runs and tests everything even if not `.rs` files have been changed
The CI scripts have a restriction, so they only run, if `.rs` files have been changed, this is good for the normal script, but prevents bors from testing and merging on the `try` and `staging` branches. Removing the restriction should be enough to make the script work again.

Noticed in #71

### PR DESCRIPTION
**Pull request by Niki4tap**
Fixes #73

### CODE REVIEW

**Comment by Niki4tap** [comment]
Uhhh, not sure if that's correct honestly :sweat_smile:

@xFrednet can you check?

File: .github/workflows/rust.yml [hunk 1]
```diff
@@ -3,25 +3,13 @@ name: Rust
 
 on:
   push:
-    # Don't run tests, when only text files were modified
```
Comment (xFrednet) [inline .github/workflows/rust.yml:6]: This is the script that normally runs for pull request and commit messages. This should still ignore the listed paths, as we don't need to run the rest if nothing changed in a commit. Also to be kind to the GH admins ^^

File: .github/workflows/rust_bors.yml [hunk 2]
```diff
@@ -3,12 +3,6 @@ name: Rust (bors)
 
 on:
   push:
-    # Don't run tests, when only text files were modified
```
Comment (xFrednet) [inline .github/workflows/rust_bors.yml:6]: This looks correct, if you revert the other file it's good to go :)

**Review by xFrednet — COMMENTED** [review]
That was fast. Small nit, but otherwise good!

File: .github/workflows/rust.yml (hunk 1 above)
Comment (Niki4tap) [reply .github/workflows/rust.yml]: Ah, thanks! Reverted

**Review by xFrednet — APPROVED** [review]

**Comment by xFrednet** [comment]
A dream, thank you very much!!!

bors r+

**Pull request merged**